src/index.js

```diff
@@ -74,7 +74,13 @@ function normalizePath(path) {
   // variable starts.
   //
   // For example if the URL is `/post/:param1::param2` we'll be escaping it to `/post/:param1\::param2`.
-  return path.replace(/{(.*?)}/g, ':$1').replace(/::/, '\\::');
+  return (
```

Comment:
Wonder if we could utilize the URL constructor here?


```
> new URL('/rating_stats?listing_ids[]=1234567', 'https://example.com').pathname
'/rating_stats'
> new URL('/people/GWID:3', 'https://example.com').pathname
'/people/GWID:3'
```

Reply:
That's not a bad idea. Will look into that after I get this fix up to the preview site.